README.rst

```diff
@@ -15,6 +15,21 @@ See other sites for the formatting and follow these rules:
 -  Include at least one screenshot.
 -  Keep the sites in alphabetical order.
 
+Ok, I'm on that list, what should I do ?
+----------------------------------------
+
+We recommend you that in the future you refer to the OWASP (Open Web Application Security Project)
+before implementing or specifying web applications.
+
+For example the current set of recommendation, and the rationals on "why" for password rules are here:
+https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Authentication_Cheat_Sheet.md
```

Comment:
Can we make this a more user friendly clickable hyperlink?

Reply:
Does it also make sense to reference the actual revised NIST guidelines, here:  https://pages.nist.gov/800-63-3/sp800-63b.html#memsecret
and the use of good meters, such as `zxcvbn`?

Afterall, it was versions of those guidelines in the past that gave s the nightmare that we see today...